code/game/gamemodes/miniantags/morph/morph.dm

```diff
@@ -144,14 +144,18 @@
 		visible_message("<span class='warning'>[src] twists and dissolves into a pile of green flesh!</span>", \
 						"<span class='userdanger'>Your skin ruptures! Your flesh breaks apart! No disguise can ward off de--</span>")
 		restore()
+	barf_contents()
 	..()
 
-/mob/living/simple_animal/hostile/morph/Destroy()
```

Comment:
Then all the stuff dies. Things should not drop their contents when qdel'd, that's like a whole meme.